src/oas3/transformers/__tests__/__snapshots__/request.test.ts.snap

```diff
@@ -4,7 +4,7 @@ exports[`translateOas3ToRequest give a request body should translate it 1`] = `
 Object {
   "body": Object {
     "contents": Array [
-      "translateMediaTypeObject({}, {\\"schema\\":{\\"deprecated\\":true}}, content-a, {\\"content-a\\":{\\"schema\\":{\\"deprecated\\":true}}})",
+      "translateMediaTypeObject([\\"content-a\\",{\\"schema\\":{\\"deprecated\\":true}}], 0, [[\\"content-a\\",{\\"schema\\":{\\"deprecated\\":true}}]])",
```

Comment:
This doesn't look quite right.

Reply:
it's correct because these are array params. That's what's given when you do `entries(media).map(translateMedia)`.
To be fair, we should probably don't mock that at all and simply print the value.
I don't really think there's any real benefit of having that small function mocked here.

Reply:
Got rid of that mock in https://github.com/stoplightio/http-spec/pull/176/commits/2fca01b189917fd66e3f23843ccc50480a8f022d